Stripe/STPAnalyticsClient.m

```diff
@@ -246,6 +264,7 @@ + (NSMutableDictionary *)commonPayload {
     payload[@"app_name"] = [NSBundle stp_applicationName];
     payload[@"app_version"] = [NSBundle stp_applicationVersion];
     payload[@"apple_pay_enabled"] = @([Stripe deviceSupportsApplePay]);
+    payload[@"card_io_enabled"] = @([STPCardIOProxy isCardIOAvailable]);
```

Comment:
Do we want to make this less card.io specific in case we support other OCR in the future? Something like key=ocr_type value=none/card_io